en/release-notes/upgrade-guide-v2.1.md

```diff
@@ -0,0 +1,164 @@
+
+# Complete list of changes (compared to v2.0.5)
+
+## What's New
+
+**3D Node**
+In __v2.1__, newly created `Node` objects are still 2D by default. When needed, you only need to click the 2.5D button at the top right of the `Node` to switch to **3D mode**. After switching, you can edit the **z axis**. See [3D Node](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/3d-node.html).
+
+You can watch converting to **3D Nodes** in action: [3D Node movie](https://www.dropbox.com/s/6wq8f0c1b6mm9dt/3D%20Node.mov?dl=0).
+
+**3D Camera**
+Like `Node`, `Camera` is also divided into two modes. When you add a `Camera` component to a normal 2D node, it becomes a 2D Camera. When you switch `Node` to **3D mode**, `Camera` will also enter **3D mode**, and then you can edit `Camera's` **FOV** and other parameters in **3D mode**.
+
+You can watch switching to **3D Camera** in action: [3D Camera movie](https://www.dropbox.com/s/gncxmsnw8gkomxl/3D%20Camera.mov?dl=0).
+
+**3D model import**
+Currently, __Cocos Creator__ supports importing 3D model formats for a very wide range of **.fbx** files. **.fbx** is the de-facto standard in 3D modeling software. The process of importing is very simple, just drag the **.fbx** model into the **Explorer** and wait a few moments for the import to complete. After the import is completed, you can see that the imported model is a folder that can be expanded in **Explorer**. When importing the model, the editor will automatically parse the content of the model and generate a **Prefab**, a **Grid**, a **Bone Animation** and other resources. See [Import Models](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/import-model.html)
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_D5EB83DE4EB14ED7F4E0412694594BEA563C6F613AB6CAC04191666891BDD83C_1543326015234_image.png)
+
+
+**Mesh Renderer Component**
+Mesh Renderer is used to draw grid resources. See [Mesh Renderer](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/mesh-renderer.html).
+
+![](https://github.com/cocos-creator/creator-docs/raw/v2.1/zh/3d/img/mesh_renderer.png)
+
+**Bone Animation Component**
+The **Skeletal Animation** component inherits from the **Animation** component. The **Skeletal Animation** component is not much different from the **Animation** component. The exception is the **Skeletal Animation** component uses a clip that can only be a **Skeletal Animation** clip. The **Skeletal Animation** component is automatically added to the model's Prefab when it is imported. See [Bone Animation Component](https://docs.cocos2d-x.org/creator/2.1/manual/en/3d/skeleton-animation.html) for details.
```

Comment:
1. It is best to have a blank line between the title and the body
2. “The **Skeletal Animation** component is not much different..." -> "The use method of the **Skeletal Animation** component is not much different..."  Maybe better?

Reply:
2. I dont understand "use method"  I modified the language based upon the word "use"